docker/dockerfiles/bedrock_base

```diff
@@ -11,11 +11,10 @@ CMD ["./docker/run-prod.sh"]
 RUN adduser --uid 1000 --disabled-password --gecos '' --no-create-home webdev
 
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends python2.7 libpython2.7 gettext build-essential libxml2-dev libxslt1.1 libxslt1-dev python-dev python-pip nodejs npm git postgresql-client libpq-dev
-RUN update-alternatives --install /usr/bin/node node /usr/bin/nodejs 10
-
-COPY ./bin/pipstrap.py /app/bin/pipstrap.py
-RUN ./bin/pipstrap.py
+    apt-get install -y --no-install-recommends \
+                    gettext build-essential python-dev python-setuptools \
```

Comment:
Yeah, it's right here! @jgmize could it be that said failed just just didn't use a newly built base image?